en/modules/cms-kit/index.md

```diff
@@ -20,6 +20,7 @@ And the following features are provided by the CMS Kit pro version:
 * [**Newsletter**](newsletter.md) system to allow users to subscribe to newsletters.
 * [**Contact form**](contact-form.md) system to allow users to write message to you.
 * [**URL forwarding**](url-forwarding.md) system to create URLs that redirect to other pages or external websites.
+* [**Poll**](poll.md) system to create polls on any resource, like a blog post or a page.
```

Comment:
Poll can be used in blog posts and pages in future version (currently now available, so should not be documented like that). However, it is only a use case. Polls are mainly independent components can be used anywhere in a website. My suggestion:

Poll system to create quick polls for users.